components/splitbutton/overview.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Overview
+page_title: SplitButton Overview
+description: Overview of the SplitButton for Blazor.
+slug: splitbutton-overview
+tags: telerik,blazor,splitbutton
+published: True
+position: 0
+---
+
+# Blazor SplitButton Overview
+
+The <a href = "https://www.telerik.com/blazor-ui/splitbutton" target="_blank">SplitButton for Blazor</a> is a combination of a button and a dropdown. It provides a collection of related user actions in compact interface. The SplitButton has one primary clickable action, which is always visible, and a list of secondary actions that are displayed in a dropdown when the user clicks on the arrow.
```

Comment:
It provides a collection of related user actions in `a` compact interface.

In this sentence, the `a` is missing.